% Template for PLoS
% Version 3.5 March 2018
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended
% to minimize problems and delays during our production
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% Once your paper is accepted for publication,
% PLEASE REMOVE ALL TRACKED CHANGES in this file
% and leave only the final text of your manuscript.
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file.
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission.
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.
%
% Do not include text that is not math in the math environment.
%
% Please add line breaks to long display equations when possible in order to fit size of the column.
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
% \usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace}
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
% \bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother



% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
%\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
%\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\today}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}






\usepackage{forarray}
\usepackage{xstring}
\newcommand{\getIndex}[2]{
  \ForEach{,}{\IfEq{#1}{\thislevelitem}{\number\thislevelcount\ExitForEach}{}}{#2}
}

\setcounter{secnumdepth}{0}

\newcommand{\getAff}[1]{
  \getIndex{#1}{University of Wisconsin-Madison,University of Illinois Urbana-Champaign}
}

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Distribution and Validation of EPSPS- and PPO-Inhibitor Resistance in
\emph{Amaranthus palmeri} from Southwestern Nebraska.} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Maxwel C Oliveira\textsuperscript{\getAff{University of Wisconsin-Madison}},
Darci A Giacomini\textsuperscript{\getAff{University of Illinois Urbana-Champaign}},
Nikola Arsenijevic\textsuperscript{\getAff{University of Wisconsin-Madison}},
Patrick J Tranel\textsuperscript{\getAff{University of Illinois Urbana-Champaign}},
Rodrigo Werle\textsuperscript{\getAff{University of Wisconsin-Madison}}\textsuperscript{*}\\
\bigskip
\textbf{\getAff{University of Wisconsin-Madison}}Department of Agronomy, University of Wisconsin-Madison, Madison,
Wisconsin, USA\\
\textbf{\getAff{University of Illinois Urbana-Champaign}}Department of Crop Sciences, University of Illinois Urbana-Champaign,
Urbana, Illinois, USA\\
\bigskip
* Corresponding author: rwerle@wisc.com\\
\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Failure to control \emph{Amaranthus palmeri} with
5-enolpyruvylshikimate-3-phosphate synthase (EPSPS)- and
protoporphyrinogen oxidase (PPO)-inhibitor herbicides was reported
across southwestern Nebraska in 2017. The objectives of this study were
to (1) confirm and validate EPSPS (glyphosate)- and PPO (fomesafen and
lactofen)-resistance in 51 \emph{A. palmeri} populations from
southwestern Nebraska using genotypic and whole-plant phenotypic assays,
and (2) determine which agronomic practices are influencing
EPSPS-inhibitor resistance in \emph{A. palmeri} populations in that
geography. Based on genotypic assay, 88\% of 51 populations contained at
least one individual with \emph{EPSPS} gene amplification
(\textgreater{} 2 copies), which confers glyphosate resistance; or a
mutation in the \emph{PPX2} gene \(\triangle\)G210 or R128G/M, endowing
PPO-inhibitor resistance in \emph{A. palmeri}. High correlation (0.83)
between genotypic and phenotypic assays demonstrated that \emph{EPSPS}
gene amplification is the main glyphosate resistance mechanism in
\emph{A. palmeri} populations from southwesten Nebraska. In contrast,
there was poor association between genotypic and phenotypic responses
for PPO resistance, which we attribute to a combination of other
resistance mechanisms being present, segregation of the populations, and
using too high of herbicide doses. Thus, genotypic assays could expedite
the process for confirmation of EPSPS- but not necessarily for
PPO-inhibitor resistance in \emph{A. palmeri} from southwestern
Nebraska. Moreover, \emph{EPSPS} gene amplification, county, current and
previous crops are the main factors influencing glyphosate resistance
within that geography. Most of EPSPS-susceptible \emph{A. palmeri}
populations were found in a few counties in high crop diversity areas.
Results presented here confirmed the spread of EPSPS- and PPO-inhibitor
resistance in \emph{A. palmeri} populations from southwestern Nebraska
and show that less diverse cropping systems are an important driver for
EPSPS-inhibitor resistance evolution in \emph{A. palmeri}.

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step.
% Author Summary not valid for PLOS ONE submissions.

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\noindent \textbf{Keywords}: Glyphosate; Fomesafen; Lactofen; Palmer
amaranth; Weed resistance.

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

\emph{Amaranthus palmeri} S. Watson (Palmer amaranth) is a plant species
indigenous to the southwestern United States (US) and northern Mexico
{[}1{]}. Despite being an edible plant used to feed animals and Native
Americans {[}2{]}, \emph{A. palmeri} has long been documented as a
serious weed problem in US cropping systems {[}3{]}. \emph{A. palmeri}
produces thousands of seeds and grows up to 2 m tall with many lateral
branches {[}4{]}, making it a very competitive species with crops
{[}5,6{]}. In the 1970s, when cotton picking became mechanized,
machinery contributed to the spread of \emph{A. palmeri} seeds across
the southern United States {[}7{]}. At that time, \emph{A. palmeri} was
considered the most successful weed of all dioecious \emph{Amaranthus}
species as it became widespread in cotton fields {[}7{]}. Currently,
\emph{A. palmeri} is the most economically damaging weed species
infesting corn, cotton, and soybean fields in the southern US {[}8,9{]}.

The economic importance of \emph{A. palmeri} is primarily related to its
ability to evolve resistance to herbicides. Factors relating to the
intrinsic biology of this species have contributed to its fast herbicide
resistance evolution {[}9{]}. For example, \emph{A. palmeri} reproduces
via obligate cross pollination, increasing the chances of hybridization
and the exchange of herbicide resistance alleles amongst
\emph{Amaranthus} species {[}10,11{]}. In addition, human-driven
selection has strongly contributed to the rise of \emph{A. palmeri} as a
problematic weed. In the US cropping systems, no-till is a standard
practice amongst growers, and \emph{A. palmeri} thrives in no-till
fields due to its small seed size, contributing to the rapid increase of
\emph{A. palmeri} individuals in field crops. Also, herbicide resistance
in \emph{A. palmeri} drastically increased when weed management
strategies shifted from the use of multiple herbicide sites of action
(SOA) in a single season to reliance on a single SOA post-emergence
herbicide (e.g., glyphosate) {[}12{]}. Thus far, \emph{A. palmeri} has
evolved resistance to herbicide resistance to multiple SOA {[}13{]},
which is a major concern as weed management in conventional US cropping
systems is largely herbicide dependent.

The history of herbicide resistance evolution in \emph{A. palmeri} is a
result of intense selection pressure from herbicides. In the 1990s, the
first documented cases of herbicide resistance were against microtubule-
{[}14{]}, ALS- {[}15{]}, and PSII-inhibitor herbicides {[}13{]}. With
the introduction of glyphosate-resistant (GR) crops, post-emergence
applications of glyphosate became widely used for weed management in
soybean, cotton, and corn, resulting in evolution of GR \emph{A.
palmeri} {[}16{]}. GR \emph{A. palmeri} spread across the southern and
midwestern US through both independent herbicide selection {[}17{]} and
seed dispersal {[}18,19{]}. The spread of GR \emph{A. palmeri} led to a
reevaluation of the use of glyphosate as a sole means of weed control
and a push to diversify weed management strategies (e.g., application of
other herbicide SOA). The use of 4-hydroxyphenylpyruvate dioxygenase
(HPPD)-, protoporphyrinogen oxidase (PPO)- and long chain fatty acids
(LCFA)- inhibitor herbicides increased in an attempt to manage GR
\emph{A. palmeri}. However, \emph{A. palmeri} populations also evolved
resistance to HPPD-, PPO- and LCFA-inhibitor herbicides {[}13{]}. New
technologies such as auxin-resistant crops may be jeopardized by the
newest reports of 2,4-D-resistant \emph{A. palmeri} {[}13{]} and the
number of \emph{A. palmeri} populations with herbicide resistance to
multiple herbicide resistance is also on the rise {[}20,21{]}.
Therefore, \emph{A. palmeri} herbicide resistance evolution is shrinking
the chemical control options for weed management in corn, soybean, and
cotton fields within US cropping systems.

In the US Midwest, corn and soybean growers strongly rely on EPSPS-
(e.g., glyphosate) and PPO- (e.g., fomesafen) inhibitor herbicides for
weed management. The recent migration of \emph{A. palmeri} into the US
Midwest poses a serious threat to the sustainability of crop production
in that geography. \emph{A. palmeri} is now overlapping territory with
another problematic dioecious \emph{Amaranthus} species, waterhemp
(\emph{Amaranthus tuberculatus}). Thus, prevention and/or rapid
diagnosis of herbicide resistance in \emph{A. palmeri} has become a
priority for agricultural stakeholders. The advances in high-throughput
genome sequencing methods are expediting the elucidation and detection
of herbicide resistance mechanisms in \emph{A. palmeri} and other weed
species. In the case of glyphosate, the most common resistance mechanism
in \emph{A. palmeri} is \emph{EPSPS} gene amplification {[}22,23{]},
while in PPO-inhibitor resistance, the major resistance mechanism is a
\emph{PPO2} glycine 210 deletion (\(\triangle\)G210) {[}24,25{]}.
Nonetheless, novel herbicide resistance mechanisms in \emph{A. palmeri}
are still being uncovered {[}22{]}, as evidenced by the recent
documentation of two mutations in the \emph{PPO2} enzyme in the R128
site of \emph{A. palmeri} {[}26{]}, and G399A, an amino acid
substitution of glycine to alanine in the catalytic domain of
\emph{PPO2} at position 399 {[}27{]}. So far, a few reports of
resistance mechanisms have been confirmed for \emph{A. palmeri}
populations with non-target-site resistance against EPSPS-{[}28{]} and
one against PPO-inhibitor herbicides {[}29{]}. Therefore, using
genotypic assays might provide faster detection of known herbicide weed
resistance mechanisms in \emph{A. palmeri}, but it fails to address
herbicide resistance resulting from novel mechanisms.

In the fall of 2017, growers in southwestern Nebraska reported failure
to control \emph{A. palmeri} with EPSPS- and PPO-inhibitor herbicides
(Werle R, personal communication). The objectives of this study were to
(1) confirm EPSPS- and PPO-inhibitor resistance in 51 \emph{A. palmeri}
populations from southwestern Nebraska via genotypic resistance assays
and compare these results via whole-plant phenotypic assay of their
progenies, and (2) evaluate agronomic practices that may contribute to
EPSPS- and PPO-inhibitor resistance in \emph{A. palmeri} populations.

\hypertarget{material-and-methods}{%
\section{Material and Methods}\label{material-and-methods}}

\hypertarget{plant-material-and-growing-conditions}{%
\subsection{Plant Material and Growing
Conditions}\label{plant-material-and-growing-conditions}}

The study was performed with 51 arbitrarily selected \emph{A. palmeri}
populations infesting cropping systems across southwestern Nebraska.
Each population was collected from a single field. Location, agronomic
practices, \emph{A. palmeri} distribution and density of each population
were collected (Table 1). In August 2017, green leaf tissues were
harvested from five actively growing plants from each of the 51 \emph{A.
palmeri} populations, then labeled and stored at -80 C to be used in
genotypic assays. Within the 51 \emph{A. palmeri} populations, a second
sampling of 19 arbitrarily selected populations was obtained by
collecting seeds (progenies) from 10 plants each in September 2017, then
cleaned and stored at 5 C until the onset of the greenhouse experiments.
Seeds were planted in 900 cm\textsuperscript{-3} plastic trays
containing potting-mix (Pro-Mix\textsuperscript \textregistered,
Quakertown, PA, USA). Emerged seedlings (1 cm) were transplanted into
164 cm\textsuperscript{-3} cone-tainers. \emph{A. palmeri} plants were
supplied with adequate water and kept under greenhouse conditions at
28/20 C day/night temperature with 80\% relative humidity. Artificial
lighting was provided using metal halide lamps (600 \(\mu\)mol
m\textsuperscript{-2} s\textsuperscript{-1}) to ensure 15 h photoperiod.

\hypertarget{genotypic-herbicide-resistance-mechanism-assays}{%
\subsection{Genotypic Herbicide Resistance Mechanism
Assays}\label{genotypic-herbicide-resistance-mechanism-assays}}

Genomic DNA extraction from leaf tissue samples collected from 51
\emph{A. palmeri} populations (five plants per population) were
performed using a modified CTAB method {[}30{]}. DNA quality and
quantity were checked on a Nanodrop 1000 (Thermo Fisher Scientific,
Inc., Waltham, MA, USA) and any samples with low DNA yields or high
protein:DNA ratios were discarded and re-extracted. TaqMan qPCR assays
were used to check for the presence of known PPO-inhibitor resistance
mutations on the \emph{PPO2} enzyme, including the glycine 210 deletion
{[}31{]} and the R128G/M mutations {[}32{]}. Samples were also tested
for glyphosate resistance via increased numbers of \emph{EPSPS} genomic
copies using a SYBR qPCR approach {[}33{]} in which \emph{EPSPS} copy
numbers were estimated based on comparison with a single-copy reference
gene (\emph{CPS}, carbamoyl phosphate synthetase).

Herein, for the objective of this study, individuals possessing
\emph{EPSPS} copy numbers \textgreater{} 2 are considered
EPSPS-resistant and individuals with presence of \(\triangle\)G210 or
R128 het mutations are considered PPO-resistant. Therefore, other
target-site and non-target site resistance mechanisms were not tested.

\hypertarget{whole-plant-phenotypic-assay}{%
\subsection{Whole-plant Phenotypic
Assay}\label{whole-plant-phenotypic-assay}}

The research was conducted under greenhouse conditions in 2018 and 2019
at the University of Wisconsin-Madison to evaluate the sensitivity of 19
\emph{A. palmeri} populations from southwestern Nebraska to EPSPS- and
PPO-inhibitor herbicides.

The experiments were conducted in a complete randomized design and the
experimental unit was a cone-tainer with a single \emph{A. palmeri}
seedling. The study was arranged in a factorial design with \emph{A.
palmeri} progenies from 19 populations and three herbicides with 20
replications and conducted twice. The arbitrarily selected 19 \emph{A.
palmeri} progenies were from Cha 3, Dun 3, Dun 4, Dun 5, Hay 1, Hay 3,
Hay 4, Kei 2, Kei 3, Kei 5, Kei 6, Log 1, Log 2, Log 4, Per 2, Per 4,
Red 2, Red 4, and Red 5 populations (Table 1). The selected herbicides
were glyphosate (Roundup PowerMAX\textsuperscript \textregistered, Bayer
Crop Science, Saint Louis, MO, US) applied at 870 g ae
ha\textsuperscript{-1} plus 2040 g ha\textsuperscript{-1} ammonium
sulfate (DSM Chemicals North America Inc., Augusta, GA); fomesafen
(Flexstar\textsuperscript \textregistered, Syngenta Crop Protection,
Greensboro, NC, USA) applied at 206 g ai ha\textsuperscript{-1} plus 0.5
L ha\textsuperscript{-1} of non-ionic surfactant
(Induce\textsuperscript \textregistered, Helena Agri-Enterprises,
Collierville, TN, USA); and lactofen
(Cobra\textsuperscript \textregistered, Valent USA LLC Agricultural
Products, Walnut Creek, CA, USA) applied at 280 g ai
ha\textsuperscript{-1} plus 0.5 L ha\textsuperscript{-1} of non-ionic
surfactant.

Herbicide treatments were applied to 8-10 cm tall \emph{A. palmeri}
plants with a single-tip chamber sprayer (DeVries Manufacturing Corp.,
Hollandale, MN, USA). The sprayer had an 8001 E nozzle (Spraying Systems
Co., North Avenue, Wheaton, IL, US) calibrated to deliver 140 L
ha\textsuperscript{-1} spray volume at 135 kPa at a speed of 2.3 km
h\textsuperscript{-1}. \emph{A. palmeri} populations were visually
assessed 21 days after treatment (DAT) as dead or alive. Plants were
considered alive when prominent green tissue was observed in growing
plants, while dead plants were completely necrotic.

\hypertarget{statistical-analysis}{%
\subsection{Statistical Analysis}\label{statistical-analysis}}

\hypertarget{genotypic-and-phenotypic-validation-of-epsps--and-ppo-inhibitor-resistance-in-a.-palmeri}{%
\subsubsection{\texorpdfstring{Genotypic and Phenotypic validation of
EPSPS- and PPO-inhibitor resistance in \emph{A.
palmeri}}{Genotypic and Phenotypic validation of EPSPS- and PPO-inhibitor resistance in A. palmeri}}\label{genotypic-and-phenotypic-validation-of-epsps--and-ppo-inhibitor-resistance-in-a.-palmeri}}

The number of EPSPS- or PPO-inhibitor-resistant \emph{A. palmeri}
individuals in the genotypic assays was converted to a percentage scale:

\emph{Equation 1}: \[G=\frac{S}{T} * 100 \] \noindent where \emph{G} is
the \% EPSPS- or PPO-inhibitor-resistant \emph{A. palmeri} individuals,
\emph{S} is the total number of \emph{A. palmeri} individuals positive
for herbicide resistance, and \emph{T} is the total number of \emph{A.
palmeri} individuals (n=3 to 5) screened for herbicide resistance in
genotypic assays. Fomesafen and lactofen are PPO-inhibitor herbicides;
thus, \emph{G} is same for both.

The number of alive \emph{A. palmeri} individuals in the phenotypic
assay were converted into a percentage scale:

\emph{Equation 2}: \[P=\frac{X}{T} * 100 \]

Where \emph{P} is the \% alive \emph{A. palmeri} individuals after
herbicide treatment in phenotypic assay (glyphosate, fomesafen, or
lactofen), \emph{X} is the total number of alive \emph{A. palmeri}
individuals 21 DAT and \emph{T} is the total number of \emph{A. palmeri}
individuals (n=40) treated with each herbicide. The P (\%) can be
determined only for 19 populations, as only 19 populations were screened
to herbicides. Data of two runs are combined.

The correlation between \emph{G} and \emph{P} for each herbicide
(glyphosate, fomesafen, and lactofen) and between the two PPO-inhibitor
herbicides, fomesafen and lactofen, were performed with Pearson's
analysis using the \emph{cor.test} function of R statistical software
version 3.6.1 {[}34{]}. The correlation value varies from -1 and 1,
where 1 is the total positive correlation, -1 the total negative
correlation, and 0 indicate no linear correlation. Pearson's analysis
tests the hypothesis that correlation between two variables is equal to
0 (null hypothesis). If \emph{P}-value \textgreater{} 0.05, the
probability is \textgreater{} 5\% that a correlation of some magnitude
between two variables could occur by chance alone assuming null
hypothesis is true; thus, no correaltion between variables. The \emph{G}
and \emph{P} correlation was performed with the 19 \emph{A. palmeri}
populations treated with herbicide in the phenotypic assay as well as
their respective genotypic assays results.

Although the individuals screened in genotypic (parent) and phenotypic
(progeny) assays are not the same; the methodology was chosen to
simulated a real farm scenario whereas growers could potentially collect
leaf samples and mail them to a laboratory for faster genotypic
herbicide resistance confirmation.

\hypertarget{random-forest}{%
\subsubsection{Random Forest}\label{random-forest}}

Random forest is a powerful ensemble machine learning algorithm which
generates and combines multiple decision trees to reach a consensus. The
random forest procedure is described in detail by Breiman {[}35{]} and
Biau {[}36{]}. In short, the random forest analysis is largely based on
two parameters: \emph{ntree}, which is the number of decision trees, and
\emph{mtry}, the number of different predictors tested in each tree. For
each decision tree, a subsample of observations from the data are
selected with replacement to train the trees (bootstrap aggregating).
These ``in-bag'' samples include approximately 66\% of the total data
and some observations may be repeated in each new training data set
since this sampling is done with replacement. The remaining 33\% of the
data are designated ``Out-of-Bag'' or OOB samples and are used in an
internal cross-validation technique to estimate the model performance
error. To evaluate the importance of an explanatory variable (or
predictor), the random forest measures both the decrease in model
performance accuracy as calculated by the OOB error and the decrease in
the Gini Index value. The Gini Index value (mean decrease in accuracy)
is the mean of a total variable decrease of a node impurity, weighted by
the samples proportion reaching that node in each individual decision
tree. Therefore, variables with a large Gini Index value indicates
higher variable importance, and are more important for data
classification. Random Forest has been used to described the incidence
of crop disease {[}37{]} and glyphosate resistance in \emph{Amaranthus}
spp. {[}38{]}.

The random forest analysis was performed with the \emph{randomForest}
package of R statistical software to describe the influence of
\emph{EPSPS} gene amplification (genotypic), PPO-inhibitor resistance
(genotypic), location (county), agronomic practices (e.g., tillage,
irrigation, current and previous cropping-system), and weed demographics
(e.g., density and distribution) on EPSPS-inhibitor resistance in
\emph{A. palmeri} in southwestern Nebraska (Table 1). \emph{EPSPS} gene
copy number (genotypic) was included as an explanatory variable to test
the robustness of random forest as it is highly correlated to glyphosate
resistance in \emph{A. palmeri} {[}22{]}. The \emph{A. palmeri}
populations with at least one individual with \textgreater{} 2
\emph{EPSPS} copy number was considered resistant (Table 2); therefore,
EPSPS-inhibitor resistance was classified as Yes (\textgreater{} 2
\emph{EPSPS} copy number) or No.~For this analysis, the \emph{ntree}
parameter was set to 500, whereas \emph{mtry} was set to 2. Parameters
numbers selected are standard for the sample size of the present study.

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{genotypic-and-phenotypic-validation-of-epsps--and-ppo-inhibitor-resistance-in-a.-palmeri-1}{%
\subsection{\texorpdfstring{Genotypic and phenotypic validation of
EPSPS- and PPO-inhibitor resistance in \emph{A.
palmeri}}{Genotypic and phenotypic validation of EPSPS- and PPO-inhibitor resistance in A. palmeri}}\label{genotypic-and-phenotypic-validation-of-epsps--and-ppo-inhibitor-resistance-in-a.-palmeri-1}}

The \emph{EPSPS} amplification was found in 63\% of the 51 \emph{A.
palmeri} populations analysed (Table 2). Phenotypic analysis of a subset
of these populations confirmed the genotypic analysis data, in that a
positive correlation (0.83; \emph{P}-value=0.0000) was observed between
\emph{G} and \emph{P} assays (Figure 1 and Table 3). Seven \emph{A.
palmeri} populations tested negative (\emph{G}=0) for glyphosate
resistance in the genotypic assay but three of these populations showed
low (\emph{P}=18\%, Hay 1), moderate (\emph{P}=35\%, Hay 4) and high
(\emph{P}=75\%, Red 5) survival after glyphosate treatment in the
phenotypic assays (Figure 1). The other four populations (Dun 3, Hay 3,
Log 2 and Kei 3) that tested negative (\emph{G}=0\%) for EPSPS-inhibitor
resistance in the genotypic assay showed less than 15\% glyphosate
survival (\emph{P}).

Fomesafen and lactofen resulted in less than 40\% survival of \emph{A.
palmeri} populations in the phenotypic assays (Figure 2). The
correlation between \emph{G} and \emph{P} for PPO-inhibitor resistance
in \emph{A. palmeri} populations was inconsistent (Table 3). While a
high \emph{G} and \emph{P} correlation (0.52; \emph{P}-value=0.0217) was
observed for fomesafen (Figure 2A), no \emph{G} and \emph{P} correlation
(-0.05; \emph{P}-value=0.84) was found for lactofen (Figure 2B).
\emph{A. palmeri} populations Dun 5, Kei 2, Kei 5, and Log 4 were
segregating for PPO resistance in the genotyping assay but individuals
in these populations were sensitive to lactofen treatment (\emph{P}=0\%,
Figure 2B). However, these populations were less sensitive to fomesafen.
For example, nearly 30\% of individuals from the Log 4 population
survived fomesafen treatment (Figure 2A). In contrast, \emph{A. palmeri}
populations Cha 3, Kei 6, Per 2, and Red 5 tested negative for
PPO-inhibitor resistance (\emph{G}=0\%) but over 15\% of populations
survived both fomesafen and lactofen treatment, suggesting the presence
of an alternate resistance mechanism. Also, \emph{A. palmeri}
populations Kei 3, Per 4, and Dun 4 showed 38, 25, and 18\% survival
after fomesafen treatment but less than 15\% for lactofen. There was no
correlation (0.23; \emph{P}-value=0.34) between fomesafen and lactofen
in the phenotypic assay (Table 3).

According to the genotypic assay, nearly 70\% of \emph{A. palmeri}
populations from southwestern Nebraska were confirmed resistant to
PPO-inhibitor herbicides. In the 34 PPO-inhibitor-resistant \emph{A.
palmeri} populations, 32 contain the \(\triangle\)G210 deletion in the
\emph{PPX2} gene, while the R128M/G endowed PPO-inhibitor resistance in
two \emph{A. palmeri} populations. Nearly 14\% of \emph{A. palmeri}
populations had all individuals resistant, 53\% were segregating for
resistance, and 33\% had no mutation detected to PPO-inhibitor
herbicides. In addition, based on \emph{EPSPS} gene amplification, our
study showed that 10\% of \emph{A. palmeri} populations had all
individuals resistant to glyphosate, 53\% were segregating for
resistance, and 37\% were susceptible to glyphosate (Table 2). Multiple
resistance (EPSPS- and PPO-inhibitor herbicides) was present in 41\% of
\emph{A. palmeri} populations of southwestern Nebraska (Table 2), while
6, 11, and 13 \emph{A. palmeri} populations were susceptible to both
herbicides, resistant to only glyphosate-, and resistant to only
PPO-inhibitor herbicides, respectively (Figure 3).

\hypertarget{random-forest-1}{%
\subsection{Random Forest}\label{random-forest-1}}

The final OOB error rate of the random forest analysis was 13.33\%,
meaning that \textgreater{} 86\% of OOB samples were adequately
classified by the model. The random forest analysis ranked (high to low)
\emph{EPSPS} gene amplification \textgreater{} county \textgreater{}
current crop \textgreater{} previous crop \textgreater{} \emph{A.
palmeri} density \textgreater{} tillage \textgreater{} irrigation
\textgreater{} \emph{A. palmeri} distribution \textgreater{}
PPO-inhibitor resistance as the factors influencing the presence of
EPSPS-inhibitor resistance in \emph{A. palmeri} of southwestern Nebraska
(Figure 4). The selection of \emph{EPSPS} gene amplification as the top
predictor shows the robustness of this model, since \emph{EPSPS} gene
amplification is known to confer resistance to glyphosate. County was
the second most important factor for the presence of EPSPS-inhibitor
resistant \emph{A. palmeri}. The presence of EPSPS-inhibitor resistance
in \emph{A. palmeri} was found in at least one population of all
surveyed counties from southwestern Nebraska (Table 2). The lowest
number of EPSPS-inhibitor-resistant populations was found at Hayes (Hay)
and Perkins (Per) County with 1 (out 5) and 2 (out 6), respectively.
Also, current and previous crops strongly influenced the presence of
EPSPS-inhibitor resistance in \emph{A. palmeri} populations. Five
\emph{A. palmeri} populations (Dun 4, Dun 5, Fro 1, Kei 2, and Lin 2)
had 100\% resistant individuals and these populations were all found in
current corn or soybean crops preceded by a corn or sorghum crops
(Figure 5A). In contrast, \emph{EPSPS} gene amplification did not occur
in 19 \emph{A. palmeri} populations, from which only two populations
were found in corn and soybean rotations (Fro 5 and Hit 5; Table 2). The
majority of EPSPS-inhibitor-susceptible \emph{A. palmeri} populations
were found in rotations of corn, soybean, sorghum, wheat, fallow, and
other crops (e.g., alfalfa, dry beans and field peas; Figure 5A).

Nearly 70\% of EPSPS-inhibitor-susceptible \emph{A. palmeri} populations
were resistant to PPO-inhibitor herbicides (Table 2). A similar trend of
crop diversity was observed for PPO-inhibitor resistance in \emph{A.
palmeri}, except a crop rotation of corn-sorghum-soybean was found in
locations with only PPO-inhibitor-resistant individuals (Figure 5B). The
random forest analysis was performed on EPSPS-inhibitor resistance only
due to the robustness of \emph{EPSPS} gene amplification as a positive
control to detect resistance in \emph{A. palmeri}, which is different
from our results with PPO-inhibitor resistance.

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

The high correlation between \emph{G} and \emph{P} for EPSPS-inhibitor
resistance demonstrates that most GR \emph{A. palmeri} populations from
southwestern Nebraska are resistant due to \emph{EPSPS} gene
amplification. However, other resistance mechanisms are likely present
since a few \emph{A. palmeri} populations showed no \emph{EPSPS} gene
amplification despite a majority of the individuals surviving glyphosate
(870 g ae ha\textsuperscript{-1}) application in the phenotypic assay.
\emph{A. palmeri} was the first identified weed to evolve glyphosate
resistance via \emph{EPSPS} gene amplification {[}39{]}, followed by
\emph{Kochia scoparia}, \emph{Amaranthus tuberculatus}, \emph{Lolium
perenne ssp. multiflorum}, \emph{Bromus diandrus}, \emph{Eleusine
indica}, \emph{Chloris truncata}, and \emph{Amaranthus hybridus}
{[}40,41{]}. The \emph{EPSPS} gene amplification mechanism is widespread
in \emph{A. palmeri} {[}22,40{]}, albeit other EPSPS-inhibitor
resistance mechanisms have arisen, including \emph{Pro106} mutation in
the \emph{EPSPS} gene and reduced glyphosate absorption/translocation
{[}28,40{]}. It remains unknown whether the \emph{A. palmeri}
populations (e.g., Red 5) which have low or no \emph{EPSPS} gene
amplification harbor additional resistance mechanisms, warranting
further investigations.

\emph{EPSPS} gene amplification is an important evolutionary mechanism
enabling weeds and other pests {[}42,43{]} to evolve resistance to
pesticides. Research on \emph{EPSPS} gene amplification molecular basis
in weed species is underway but more work is still needed to unveil this
complex adaptative trait. The genetics of \emph{EPSPS} gene
amplification in weed species follows Mendelian (\emph{K. scoparia})
{[}44{]} and non-Mendelian (\emph{A. palmeri} {[}45{]} and \emph{B.
diandrus} {[}46{]}) inheritance patterns. Over 100 \emph{EPSPS} gene
copies have been documented in \emph{A. palmeri} but maximum of only 10
have been observed in \emph{K. scoparia} {[}47,48{]}. The \emph{EPSPS}
gene copy variation in \emph{A. palmeri} is a result of the
extrachromosomal circular DNA transmitted to the next generation by
tethering to mitotic and meiotic chromosomes (eccDNA) {[}49{]}, while in
\emph{K. scoparia}, \emph{EPSPS} copies are arranged in tandem repeats
at a single locus {[}41{]}. Segregation for \emph{EPSPS} copy number in
\emph{A. palmeri} families (F\scriptsize 1 \normalsize and
F\scriptsize 2 \normalsize) is transgressive, with individuals varying
in \emph{EPSPS} gene amplification levels even among clonal plants
{[}45{]}. Transgressive segregation for \emph{EPSPS} in \emph{A.
palmeri} might explain the variable \emph{EPSPS} copy numbers between
individuals within populations screened from southwestern Nebraska
(Table 2). Gene amplification coupled with the dioecious nature of
\emph{A. palmeri} are valuable traits for this weed that help increase
its genetic complexity and allow it to adapt to current US cropping
systems.

According to the genotypic assay, most of \emph{A. palmeri} populations
from southwestern Nebraska also showed resistance to PPO-inhibitor
herbicides. The mechanism of resistance is largely due to the
\(\triangle\)G210 deletion but also due to R128M/G mutations in the
\emph{PPX2} gene (Table 2). It has been demonstrated that a mutated PPO
enzyme has reduced affinity for several PPO-inhibitor herbicides in
\emph{A. palmeri} {[}20{]}. Application of fomesafen (206 g ai
ha\textsuperscript{-1}) and lactofen (280 g ai ha\textsuperscript{-1})
provided high mortality in \emph{A. palmeri} populations from
southwestern Nebraska, including \emph{A. palmeri} populations with
100\% individuals with \(\triangle\)G210 deletion. Therefore, it is
possible that most of sprayed \emph{A. palmeri} individuals in the
phenotypic assay had no mutation for PPO-inhibitor resistance. High
\emph{A. palmeri} mortality suggests that populations are segregating
for PPO-inhibitor resistance at a low level. Still, there is correlation
between \emph{G} and \emph{P} for fomesafen but not lactofen, which was
sprayed slightly above the labled rate of 218 g ai
ha\textsuperscript{-1}. The validation for PPO-inhibitor resistance
presented here is limited by the segregating nature of \emph{A. palmeri}
collected from field populations, since individuals in the phenotypic
assays were not the same as those screened in the genotypic assays. This
could be also explained in several ways, 1) the number of individuals
sampled for the \emph{G} assay study may have been too low for the
objective of validation; 2) high herbicide rate resulting in high
individual mortality; 3) greenhouse conditions were ideal and plants had
no stress during application of PPO-inhibitor herbicides, which might be
different from plants under field conditions of southwestern Nebraska
{[}50{]}; and 4) it is likely that plants were smaller than usual
because of small volume of cone-tainers, the size of a plant strongly
impacts the level of resistance, with smaller plants showing less
resistance than larger plants {[}51{]}.

Populations of \emph{A. palmeri} have been reported to be resistant to
HPPD-{[}52{]}, PSII-{[}52{]} and EPSPS-inhibitor herbicides {[}53{]} in
Nebraska. In 2014, a survey with \emph{Amaranthus} spp. in southwestern
Nebraska showed widespread EPSPS-inhibitor resistance for \emph{A.
tuberculatus} (81\%) but not for \emph{A. palmeri} (6\%) {[}38{]}. Our
survey showed a high number of EPSPS-inhibitor-resistant/segregating
\emph{A. palmeri} populations in southwestern Nebraska
(\textgreater60\%). The rapid evolution of EPSPS-inhibitor-resistant
\emph{A. palmeri} in southwestern Nebraska raised questions about
whether populations were introduced via seed/gene flow or arose
independently. Although we did not test the hypothesis, the random
forest analysis did shed some light on EPSPS-inhibitor resistance
evolution in \emph{A. palmeri}. The random forest analysis results
suggested that \emph{EPSPS} gene amplification, county, and current and
previous crops were the top factors influencing GR \emph{A. palmeri} in
southwestern Nebraska. The \emph{EPSPS} gene amplification as the
primary factor for glyphosate resistance in \emph{A. palmeri} showed the
robustness of random forest analysis, while county influence on
\emph{EPSPS}-inhibitor resistance in \emph{A. palmeri} highlighted how
practices for delaying EPSPS-inhibitor resistance evolution should start
at the landscape level. It is likely that growers with common goals may
implement best weed management tactics to delay EPSPS-inhibitor
resistance evolution and prevent seed/gene flow of herbicide resistant
weeds. Nonetheless, it is striking that PPO-inhibitor resistance was
common in counties with low/no EPSPS-inhibitor resistance. For instance,
most EPSPS-inhibitor-susceptible \emph{A. palmeri} were found in Hayes
County in areas of sorghum or corn in rotation with wheat (Table 1),
which are less glyphosate-dependent crops. Although best weed management
practices at the landscape level is encouraged, our results suggest that
growers were not necessarily implementing good management practices but
rather, using less EPSPS-inhibitor herbicide.

The high presence of \emph{A. palmeri} populations with 100\% resistance
to EPSPS-inhibitors in less diverse cropping systems (e.g., corn-soybean
rotation) suggests the influence of repeated glyphosate application on
\emph{A. palmeri} evolution to EPSPS-inhibitor resistance. The
occurrence of EPSPS-inhibitor-resistant \emph{A. palmeri} populations is
reduced in rotations with more diversified crops (Figure 5A), most
likely due to rotations with glyphosate sensitive crops (no/less
glyphosate use). Crop diversity exerts a different selection pressure on
weed communities, including canopy closure timing, seeding, and harvest
date, which aid on reducing the dominance of single weed species
{[}54{]}. Nonetheless, it is likely that Nebraska growers still rely on
other herbicide sites of action for weed management, as most
EPSPS-inhibitor-susceptible \emph{A. palmeri} populations were resistant
to PPO-inhibitor herbicides (e.g., fomesafen). Similar to
EPSPS-inhibitor resistance, the PPO-inhibitor-resistant \emph{A.
palmeri} was also found more often in less diverse crop rotations
(Figure 5B). Despite not having long-term herbicide application records
for the surveyed area, it is possible that overreliance on a single or
few herbicide SOA in areas with low crop diversity contributed to
resistance. In addition, it has been shown that herbicide mixture
(multiple SOA in one application) is more effective for delaying
herbicide weed resistance than herbicide rotation (multiple
applications, each with a single SOA) {[}55,56{]}. Thus, it is likely
that without herbicide mixture, increased crop diversity is not enough
to reduce the risk of herbicide resistance evolution in weed species.

An emerging concern in weed science is the ability of some species to
stack genes for multiple herbicide resistance in a single population.
Five- and six-way herbicide resistance has been reported in an \emph{A.
tuberculatus} population from Illinois {[}57{]} and Missouri {[}58{]},
respectively. The 19 \emph{A. palmeri} populations used in the
whole-plant phenotypic also tested positive (\textgreater80\% of
individuals) to an ALS-inhibitor herbicide (imazethapyr {[}70 g ai
ha\textsuperscript{-1}{]}; data not shown). Moreover, nearly half of the
\emph{A. palmeri} populations in this study possessed multiple herbicide
resistance (EPSPS and PPO; Figure 3). Thus, it is likely that two- and
three-way resistance exists in most \emph{A. palmeri} populations from
southwestern Nebraska. The random forest analysis suggested no/low
influence of PPO-inhibitor resistance on EPSPS-inhibitor resistance,
indicating \emph{EPSPS} gene amplification and PPO-inhibitor resistance
mutations (\(\triangle\)G210 or R128M/G) are not genetically linked.
Cosegregation of distinct SOA resistance mechanisms is unlikely but not
impossible as it has been shown that ALS- and PPO-inhibitor resistance
are genetically linked in a \emph{A. tuberculatus} population {[}59{]}.

Herein we demonstrate the widespread occurrence of EPSPS- and
PPO-inhibitor-resistant \emph{A. palmeri} in southwestern Nebraska.
Glyphosate resistance via \emph{EPSPS} gene amplification was present in
the majority of \emph{A. palmeri} populations from Nebraska, supporting
the use of genotypic assays for faster detection of
\emph{EPSPS}-inhibitor resistance in this region. Novel or non-target
glyphosate resistance mechanisms may also be present in few \emph{A.
palmeri} populations, but at a relatively low frequency. PPO-inhibitor
resistance was also present in these populations, but was less
correlated with known PPO-inhibitor resistance mutations, warranting the
use of both genotypic and whole-plant bioassays for testing
PPO-inhibitor resistance. The EPSPS- or PPO-inhibitor-resistant \emph{A.
palmeri} populations were found in areas of low crop diversity,
suggesting resistance evolution was mostly due to high selection
pressures of EPSPS- and PPO-inhibitor herbicides. Great progress has
been made towards understanding the \emph{A. palmeri} molecular basis of
resistance, but the continuous spread of herbicide resistance in
\emph{A. palmeri} populations to new geographies is evident. Thus, there
is a need to change the current methods for crop and weed management,
including dependence on corn/soybean rotation, and bring non-herbicide
weed management innovations to US cropping systems.

\hypertarget{acknowledgments}{%
\section{Acknowledgments}\label{acknowledgments}}

This research received no specific grant from any funding agency. We
also appreciate the help of Victor Ribeiro's assistance with the
greenhouse projects. No conflicts of interest have been declared.

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\begin{table}[!ht]  \small
\begin{adjustwidth}{-2.5in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Demographic list of \textit{Amaranthus palmeri} populations with respective Nebraska County location and agronomic practices}}
\begin{tabular}{|l+c|c|c|c|c|c|c|c|}
\hline
{\bf Population}  & {\bf County} & {\bf Current crop} & {\bf Previous crop$^a$} & {\bf Tillage} & {\bf Irrigation} & {\bf Weed distribution} & {\bf Weed density}\\ \thickhline
Cha 1 & Chase & sorghum  & corn & tilled & rainfed & spread & Low\\ \hline
Cha 2 & Chase   & corn & wheat  & strip-till & center pivot &   spread & high\\ \hline
Cha 3   & Chase & corn &    fallow/cornstalks   & no-till   & centerpivot   & spread &  high\\ \hline
Cha 4   & Chase & soybeans &    fallow/cornstalks   & no-till   & centerpivot   & spread &  low\\ \hline
Cha 5   & Chase & corn &    corn    & strip-till    & rainfed & spread &    low\\ \hline
Dun 1   & Dundy & wheatstubble &    other$^b$ & tilled &    rainfed &   spread &    intermediate\\ \hline
Dun 2   & Dundy & corn &    sorghum &   no-till &   rainfed &   spread &    intermediate\\ \hline
Dun 3   & Dundy &   other   &    & tilled & centerpivot &   spread &    intermediate\\ \hline
Dun 4   & Dundy & corn &    corn &  no-till &   centerpivot &   edges & high\\ \hline
Dun 5   & Dundy &   soybeans &  corn &  tilled &    centerpivot &   spread &    low\\ \hline
Fro 1   & Frontier &    corn &  sorghum & no-till   & rainfed & edges & high\\ \hline
Fro 2   & Frontier &    soybeans &  corn &  tilled &    rainfed &   edges   & low\\ \hline
Fro 3   & Frontier &    soybeans &  wheatstubble &  tilled &    centerpivot & spread &  high\\ \hline
Fro 4   & Frontier &    sorghum  & fallow/cornstalks &  tilled &    rainfed  & edges &  intermediate\\ \hline
Fro 5   & Frontier &    soybeans &  corn &  tilled &    centerpivot & edges & high\\ \hline
Hay 1   & Hayes & sorghum & fallow/cornstalks    & tilled & centerpivot  & spread & intermediate\\ \hline
Hay 2   & Hayes & corn &    wheatstubble &  no-till &   rainfed  & spread & intermediate\\ \hline
Hay 3   & Hayes & sorghum    & wheatstubble & tilled &  centerpivot &   spread &    high\\ \hline
Hay 4   & Hayes & corn &    wheatstubble &  no-till &   rainfed  & edges &  intermediate\\ \hline
Hay 5   & Hayes & sorghum & wheatstubble &  no-till &   rainfed &   spread &    high\\ \hline
Hit 1   & Hitchcock & corn &    fallow/cornstalks & tilled &    centerpivot &   edges & low\\ \hline
Hit 2   & Hitchcock & soybeans &    corn &  no-till &   rainfed &   spread &    low\\ \hline
Hit 3   & Hitchcock & corn &    corn &  no-till & rainfed   & edges & high\\ \hline
Hit 4 & Hitchcock   & sorghum & wheatstubble &  no-till &   rainfed &   edges & high\\ \hline
Hit 5   & Hitchcock & soybeans &    corn    & no-till & centerpivot &   edges   & high\\ \hline
Kei 1   & Keith & other &   fallow/cornstalks & tilled &    centerpivot &   spread &    high\\ \hline
Kei 2   & Keith & corn &    fallow/cornstalks & no-till &   centerpivot  & spread & intermediate\\ \hline
Kei 3   & Keith & soybeans &    &   tilled &    furrow &    spread &    high\\ \hline
Kei 4   & Keith & soybeans &      & no-till &   centerpivot  & spread & low\\ \hline
Kei 5   & Keith & other &   corn &  tilled  & centerpivot   & spread &  low\\ \hline
Kei 6   & Keith & soybeans &    &   no-till &   centerpivot &   spread &    high\\ \hline
Lin 1   & Lincoln & corn &  other & no-till & centerpivot & spread &    high\\ \hline
Lin 2   & Lincoln     & soybeans &  corn &  tilled &    centerpivot &   spread  & low\\ \hline
Lin 3   & Lincoln   & soybeans &    &   tilled &    centerpivot &   spread &    low\\ \hline
Lin 4   & Lincoln   & corn &    &   tilled &    furrow &    spread &    high\\ \hline
Lin 5   & Lincoln   & corn &    wheatstubble     & no-till &    rainfed &   spread &    high\\ \hline
Log 1   & Logan  & soybeans &   fallow/cornstalks & tilled &    centerpivot &   edges & intermediate\\ \hline
Log 2   & Logan  & other &  fallow/cornstalks    & no-till &    rainfed  & spread & intermediate\\ \hline
Log 3   & Logan &   soybeans &  corn &  tilled &    centerpivot &   edges & high\\ \hline
Log 4   & Logan &   soybeans &  corn &  tilled &    rainfed  & spread & low\\ \hline
Per 1   & Perkins & other & sorghum &   no-till &   rainfed & spread &  low\\ \hline
Per 2   & Perkins & soybeans &  corn &  strip-till &    centerpivot &   spread &    intermediate\\ \hline
Per 3   & Perkins & fallow/cornstalks & corn &  tilled &    rainfed &   spread &    high\\ \hline
Per 4   & Perkins & soybeans &  corn &  no-till &   centerpivot &   spread &    high\\ \hline
Per 5   & Perkins & other    & fallow/cornstalks &  no-till &   centerpivot &   spread &    intermediate\\ \hline
Per 6   & Perkins   & other & fallow/cornstalks &   no-till  & centerpivot &    spread &    high\\ \hline
Red 1   & Red Willow &  soybeans &  corn &  no-till & centerpivot & edges & high\\ \hline
Red 2   & Red Willow &  corn &  corn &  tilled &    centerpivot &   edges & high \\ \hline
Red 3   & Red Willow &  wheatstubble &  wheat    & no-till &    rainfed &   spread &    intermediate\\ \hline
Red 4   & Red Willow &  corn &  corn &  no-till &   rainfed & spread &  low\\ \hline
Red 5   & Red Willow &  fallow/cornstalks & corn    & no-till & rainfed &   spread &    high\\ \hline
\end{tabular}
\begin{flushleft}
\end{flushleft}
\label{table1}
\footnotesize{$^a$ Blank cells means unidentified crop, $^b$alfalfa, dry beans and field peas}\\
\end{adjustwidth}
\end{table}

\begin{table}[!ht]  \small
\begin{adjustwidth}{-2.5in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf List of \textit{Amaranthus palmeri} populations with EPSPS gene amplification and PPO resistance according to genotypic resistance assays.}}
\begin{tabular}{|l+c|c|c|p{1.8cm}|c|p{1.8cm}|c|}
\thickhline
\multicolumn{1}{|c|}{\bf population} & \multicolumn{4}{c}{\bf EPSPS gene amplification}  & \multicolumn{2}{|c|}{\bf PPO resistance$^a$}  & \multicolumn{1}{c|}{\bf \# Plants$^b$}\\ 
\cline{2-5}\cline{6-7}
{\bf }  & {\bf Mean} & {\bf Max.} &  {\bf Min.} & {\bf \% EPSPS resistant plants} & {\bf Mutation} & {\bf \% PPO resistant plants} & {\bf }\\ \thickhline
Cha 1 & 7 & 23 & 1 & 25 &  & 0 & 4\\ \hline
Cha 2 & 1 & 3 & 1   & 20 &  & 0 & 5\\ \hline
Cha 3   & 9 & 15 & 1 & 80   &  & 0 & 5\\ \hline
Cha 4   & 10 & 26 & 1   & 40 & R128 het & 20 & 5\\ \hline
Cha 5   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Dun 1   & 5 & 18 & 1    & 60 & $\triangle$G210 & 20 & 5\\ \hline
Dun 2   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Dun 3   & 1 & 1 & 1 & 0 & $\triangle$G210 & 67 & 3\\ \hline
Dun 4   & 6 & 10 & 4 & 100 &  & 0 & 5\\ \hline
Dun 5   & 24 & 51 & 4 & 100 &   $\triangle$G210 & 20 & 5\\ \hline
Fro 1   & 6 &   10 & 3 & 100 & $\triangle$G210 & 33 & 3\\ \hline
Fro 2   & 3 & 6 & 1 & 33 & $\triangle$G210 & 100 & 3\\ \hline
Fro 3   & 5 & 11 & 1 & 33 & $\triangle$G210 & 67 & 3 \\ \hline
Fro 4   & 1 & 1 & 1 & 0 &   $\triangle$G210 & 100 & 3\\ \hline
Fro 5   & 1 & 2 & 1 & 0 &    & 0 & 5\\ \hline
Hay 1   & 1 & 1 & 1 & 0 & $\triangle$G210 & 100 & 3\\ \hline
Hay 2   & 2 & 3 & 1 & 33 & $\triangle$G210  & 33 & 3\\ \hline
Hay 3   & 2 & 2 & 1 & 0 & $\triangle$G210 & 100 & 3\\ \hline
Hay 4   & 1 & 1 & 1 & 0 &   $\triangle$G210 & 67 & 3\\ \hline
Hay 5   & 1 & 1 & 1 & 0 &  & 0 & 5\\ \hline
Hit 1   & 5 & 20 & 1 & 20 &  & 0 & 5\\ \hline
Hit 2   & 21    & 57 & 3 & 67 &  & 0 & 3\\ \hline
Hit 3   & 3 & 6 & 1 & 33 & $\triangle$G210 & 33 & 3\\ \hline
Hit 4 & 2   & 3 & 1 & 25 &  & 0 & 4\\ \hline
Hit 5   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Kei 1   & 13    & 38 & 1 & 33 & $\triangle$G210 & 33 & 3\\ \hline
Kei 2   & 12    & 19 & 7 & 100 & $\triangle$G210 & 33 & 3\\ \hline
Kei 3   & 1 & 1 & 1 & 0 &  & 0 & 5\\ \hline
Kei 4   & 8 & 18 & 1 & 60 &  & 0 &  5\\ \hline
Kei 5   & 5 & 8 & 1 & 67 & $\triangle$G210 & 67 & 3\\ \hline
Kei 6   & 17    & 40 & 2 & 80 &  & 0 & 5\\ \hline
Lin 1   & 1 & 2 & 1 & 0 & $\triangle$G210 & 100 & 3\\ \hline
Lin 2   & 5 & 6 & 3 & 100 & $\triangle$G210 & 67 & 3\\ \hline
Lin 3   & 4 & 6 & 1 & 67 & $\triangle$G210 & 100 & 3\\ \hline
Lin 4   & 3 & 6 & 1 & 33 & $\triangle$G210 & 100 & 3\\ \hline
Lin 5   & 1 & 1 & 1 & 0 &  & 0 & 5\\ \hline
Log 1   & 34    & 57 & 1 & 67 & $\triangle$G210 & 33 &  3\\ \hline
Log 2 & 1   & 1 & 1 & 0 &  & 0 & 3\\ \hline
Log 3   & 4 & 7 & 1 & 67 & $\triangle$G210 & 33 & 3\\ \hline
Log 4   & 3 & 6 & 1 & 67 & $\triangle$G210 & 67 & 3\\ \hline
Per 1   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Per 2   & 32    & 59 & 1 & 80 &  & 0 & 5\\ \hline
Per 3   & 1 & 1 & 1 & 0 & $\triangle$G210 & 33 & 3\\ \hline
Per 4   & 10    & 22 & 1 & 67 &  & 0 &  3\\ \hline
Per 5   & 1 & 2 & 1 & 0 & $\triangle$G210 & 33 &  3\\ \hline
Per 6   & 1 & 2 & 1 & 0 & $\triangle$G210 & 33 &  3\\ \hline
Red 1   & 2 & 3 & 1 & 33 & $\triangle$G210 & 33 &  3\\ \hline
Red 2   & 2 & 3 & 1 & 33 & $\triangle$G210 & 67 &  3\\ \hline
Red 3   & 2 & 6 & 1 & 20 & R128 het & 20 &  5\\ \hline
Red 4   & 2 & 5 & 1 & 33 & $\triangle$G210 & 67 &  3\\ \hline
Red 5   & 1 & 2 & 1 & 0 &  & 0 &  5\\ \hline
\end{tabular}
\begin{flushleft}
\end{flushleft}
\label{table1}
\footnotesize{$^a$ blank cells means no PPO resistance found, $^b$ Number of plants screened in the genotypic herbicide resistance assay}\\
\end{adjustwidth}
\end{table}

\begin{table}[!ht]
\begin{adjustwidth}{-2.5in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Correlation estimates between genotypic (\textit{G}) and whole-plant phenotypic (\textit{P}) assays of glyphosate, fomesafen, lactofen, and between \textit{P} fomesafen and \textit{P} lactofen (PPO inhibitors).}}
\begin{tabular}{|l+c|c|c|c|c|c|}
\hline
{\bf Herbicide}  & {\bf Correlation variables} & {\bf Estimate} & {\bf lower CI$^a$} & {\bf upper CI} & {\bf t} & {\bf \textit{P}-value} \\ \thickhline
glyphosate & \textit{G} and \textit{P} & 0.83 & 0.60  & 0.93 & 6.15 & 0.0000\\ \hline
fomesafen &  \textit{G} and \textit{P}    & 0.52    & 0.09 & 0.79   & 2.53 & 0.0217\\ \hline
lactofen    &  \textit{G} and \textit{P}    & -0.05 & -0.49 &   0.41    & -0.20 & 0.8412\\ \hline
PPO inhibitors  & \textit{P}-fomesafen and \textit{P}-lactofen   & 0.23 & -0.25 &   0.62 & 0.98 & 0.3428\\ \hline
\end{tabular}
\begin{flushleft}
\end{flushleft}
\label{table1}
\footnotesize{$^a$ Confidence interval}\\
\end{adjustwidth}
\end{table}

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-sauer_recent_1957}{}%
1. Sauer J. Recent migration and evolution of the dioecious Amaranths.
Evolution. 1957;11: 11--31.
doi:\href{https://doi.org/10.2307/2405808}{10.2307/2405808}

\leavevmode\hypertarget{ref-smith_fodder_1900}{}%
2. Smith J. Fodder and forage plants. Exclusive of the grasses.
Washington; 1900.

\leavevmode\hypertarget{ref-hamilton_weeds_1958}{}%
3. Hamilton KC, Arle HF. Weeds od crops in Southern Arizona. Tucson, AZ:
College of Agriculture, University of Arizona; 1958.

\leavevmode\hypertarget{ref-sauer_revision_1955}{}%
4. Sauer J. Revision of the dioecious Amaranths. Madroo. 1955;13:
5--46. Available: \url{https://www.jstor.org/stable/41422838}

\leavevmode\hypertarget{ref-morgan_competitive_2001}{}%
5. Morgan GD, Baumann PA, Chandler JM. Competitive impact of Palmer
amaranth (\emph{Amaranthus} \emph{Palmeri}) on cotton (\emph{Gossypium}
\emph{Hirsutum}) development and yield. Weed Technology. 2001;15:
408--412.
doi:\href{https://doi.org/10.1614/0890-037X(2001)015\%5B0408:CIOPAA\%5D2.0.CO;2}{10.1614/0890-037X(2001)015{[}0408:CIOPAA{]}2.0.CO;2}

\leavevmode\hypertarget{ref-massinga_water_2003}{}%
6. Massinga RA, Currie RS, Trooien TP. Water use and light interception
under Palmer amaranth (\emph{Amaranthus} \emph{Palmeri}) and corn
competition. Weed Science. 2003;51: 523--531.
doi:\href{https://doi.org/10.1614/0043-1745(2003)051\%5B0523:WUALIU\%5D2.0.CO;2}{10.1614/0043-1745(2003)051{[}0523:WUALIU{]}2.0.CO;2}

\leavevmode\hypertarget{ref-sauer_dioecious_1972}{}%
7. Sauer JD. The dioecious Amaranths: A new species name and major range
extensions. Madroo. 1972;21: 426--434. Available:
\url{https://www.jstor.org/stable/41423815}

\leavevmode\hypertarget{ref-price_glyphosate-resistant_2011}{}%
8. Price AJ, Balkcom KS, Culpepper SA, Kelton JA, Nichols RL, Schomberg
H. Glyphosate-resistant Palmer amaranth: A threat to conservation
tillage. Journal of Soil and Water Conservation. 2011;66: 265--275.
doi:\href{https://doi.org/10.2489/jswc.66.4.265}{10.2489/jswc.66.4.265}

\leavevmode\hypertarget{ref-ward_palmer_2013}{}%
9. Ward SM, Webster TM, Steckel LE. Palmer Amaranth (\emph{Amaranthus}
\emph{Palmeri}): A Review. Weed Technology. 2013;27: 12--27.
doi:\href{https://doi.org/10.1614/WT-D-12-00113.1}{10.1614/WT-D-12-00113.1}

\leavevmode\hypertarget{ref-oliveira_interspecific_2018}{}%
10. Oliveira MC, Gaines TA, Patterson EL, Jhala AJ, Irmak S, Amundsen K,
et al. Interspecific and intraspecific transference of metabolism-based
mesotrione resistance in dioecious weedy \emph{Amaranthus}. The Plant
Journal. 2018;96: 1051--1063.
doi:\href{https://doi.org/10.1111/tpj.14089}{10.1111/tpj.14089}

\leavevmode\hypertarget{ref-gaines_interspecific_2012}{}%
11. Gaines TA, Ward SM, Bukun B, Preston C, Leach JE, Westra P.
Interspecific hybridization transfers a previously unknown glyphosate
resistance mechanism in \emph{Amaranthus} species. Evolutionary
Applications. 2012;5: 29--38.
doi:\href{https://doi.org/10.1111/j.1752-4571.2011.00204.x}{10.1111/j.1752-4571.2011.00204.x}

\leavevmode\hypertarget{ref-powles_evolved_2008}{}%
12. Powles SB. Evolved glyphosate-resistant weeds around the world:
Lessons to be learnt. Pest Management Science. 2008;64: 360--365.
doi:\href{https://doi.org/10.1002/ps.1525}{10.1002/ps.1525}

\leavevmode\hypertarget{ref-heap_list_2019-1}{}%
13. Heap I. List of Herbicide Resistant Weeds by Weed Species. 2019.
Available: \url{http://www.weedscience.org/Summary/Species.aspx}

\leavevmode\hypertarget{ref-gossett_resistance_1992}{}%
14. Gossett BJ, Murdock EC, Toler JE. Resistance of Palmer amaranth
(\emph{Amaranthus} \emph{Palmeri}) to the dinitroaniline herbicides.
Weed Technology. 1992;6: 587--591. Available:
\url{https://www.jstor.org/stable/3987215}

\leavevmode\hypertarget{ref-horak_biotypes_1995}{}%
15. Horak MJ, Peterson DE. Biotypes of Palmer amaranth
(\emph{Amaranthus} \emph{Palmeri}) and common waterhemp
(\emph{Amaranthus} \emph{Rudis}) are resistant to imazethapyr and
thifensulfuron. Weed Technology. 1995;9: 192--195.
doi:\href{https://doi.org/10.1017/S0890037X00023174}{10.1017/S0890037X00023174}

\leavevmode\hypertarget{ref-culpepper_glyphosate-resistant_2006}{}%
16. Culpepper AS, Grey TL, Vencill WK, Kichler JM, Webster TM, Brown SM,
et al. Glyphosate-resistant Palmer amaranth (\emph{Amaranthus}
\emph{Palmeri}) confirmed in Georgia. Weed Science. 2006;54: 620--626.
doi:\href{https://doi.org/10.1614/WS-06-001R.1}{10.1614/WS-06-001R.1}

\leavevmode\hypertarget{ref-kupper_population_2018}{}%
17. Kpper A, Manmathan HK, Giacomini D, Patterson EL, McCloskey WB,
Gaines TA. Population genetic structure in glyphosate-resistant and
-susceptible Palmer amaranth (\emph{Amaranthus} \emph{Palmeri})
populations using genotyping-by-sequencing (GBS). Front Plant Sci.
2018;9.
doi:\href{https://doi.org/10.3389/fpls.2018.00029}{10.3389/fpls.2018.00029}

\leavevmode\hypertarget{ref-farmer_evaluating_2017}{}%
18. Farmer JA, Webb EB, Pierce RA, Bradley KW. Evaluating the potential
for weed seed dispersal based on waterfowl consumption and seed
viability. Pest Management Science. 2017;73: 2592--2603.
doi:\href{https://doi.org/10.1002/ps.4710}{10.1002/ps.4710}

\leavevmode\hypertarget{ref-norsworthy_-field_2014}{}%
19. Norsworthy JK, Griffith G, Griffin T, Bagavathiannan M, Gbur EE.
In-field movement of glyphosate-resistant Palmer amaranth
(\emph{Amaranthus} \emph{Palmeri}) and its impact on cotton lint Yield:
Evidence supporting a zero-threshold strategy. Weed Science. 2014;62:
237--249.
doi:\href{https://doi.org/10.1614/WS-D-13-00145.1}{10.1614/WS-D-13-00145.1}

\leavevmode\hypertarget{ref-schwartz-lazaro_resistance_2017}{}%
20. Schwartz-Lazaro LM, Norsworthy JK, Scott RC, Barber LT. Resistance
of two Arkansas Palmer amaranth populations to multiple herbicide sites
of action. Crop Protection. 2017;96: 158--163.
doi:\href{https://doi.org/10.1016/j.cropro.2017.02.022}{10.1016/j.cropro.2017.02.022}

\leavevmode\hypertarget{ref-kumar_confirmation_2019}{}%
21. Kumar V, Liu R, Boyer G, Stahlman PW. Confirmation of 2,4-D
resistance and identification of multiple resistance in a Kansas Palmer
amaranth (\emph{Amaranthus} \emph{Palmeri}) population. Pest Management
Science. 2019;0.
doi:\href{https://doi.org/10.1002/ps.5400}{10.1002/ps.5400}

\leavevmode\hypertarget{ref-gaines_molecular_2019}{}%
22. Gaines TA, Patterson EL, Neve P. Molecular mechanisms of adaptive
evolution revealed by global selection for glyphosate resistance. New
Phytologist. 2019;0.
doi:\href{https://doi.org/10.1111/nph.15858}{10.1111/nph.15858}

\leavevmode\hypertarget{ref-gaines_mechanism_2011}{}%
23. Gaines TA, Shaner DL, Ward SM, Leach JE, Preston C, Westra P.
Mechanism of Resistance of Evolved Glyphosate-Resistant Palmer Amaranth
(\emph{Amaranthus} \emph{Palmeri}). J Agric Food Chem. 2011;59:
5886--5889.
doi:\href{https://doi.org/10.1021/jf104719k}{10.1021/jf104719k}

\leavevmode\hypertarget{ref-salas_resistance_2016}{}%
24. Salas RA, Burgos NR, Tranel PJ, Singh S, Glasgow L, Scott RC, et al.
Resistance to PPO-inhibiting herbicide in Palmer amaranth from Arkansas.
Pest Management Science. 2016;72: 864--869.
doi:\href{https://doi.org/10.1002/ps.4241}{10.1002/ps.4241}

\leavevmode\hypertarget{ref-salas-perez_frequency_2017}{}%
25. Salas-Perez RA, Burgos NR, Rangani G, Singh S, Refatti JP, Piveta L,
et al. Frequency of gly-210 deletion mutation among protoporphyrinogen
oxidase inhibitor--resistant Palmer amaranth (\emph{Amaranthus}
\emph{Palmeri}) populations. Weed Science. 2017;65: 718--731.
doi:\href{https://doi.org/10.1017/wsc.2017.41}{10.1017/wsc.2017.41}

\leavevmode\hypertarget{ref-giacomini_two_2017-1}{}%
26. Giacomini DA, Umphres AM, Nie H, Mueller TC, Steckel LE, Young BG,
et al. Two new PPX2 mutations associated with resistance to
PPO-inhibiting herbicides in \emph{Amaranthus} \emph{Palmeri}. Pest
Management Science. 2017;73: 1559--1563.
doi:\href{https://doi.org/10.1002/ps.4581}{10.1002/ps.4581}

\leavevmode\hypertarget{ref-rangani_novel_2019}{}%
27. Rangani G, Salas-Perez RA, Aponte RA, Knapp M, Craig IR, Mietzner T,
et al. A Novel Single-Site Mutation in the Catalytic Domain of
Protoporphyrinogen Oxidase IX (PPO) Confers Resistance to PPO-Inhibiting
Herbicides. Front Plant Sci. 2019;10.
doi:\href{https://doi.org/10.3389/fpls.2019.00568}{10.3389/fpls.2019.00568}

\leavevmode\hypertarget{ref-dominguez-valenzuela_first_2017}{}%
28. Dominguez-Valenzuela JA, Gherekhloo J, Fernndez-Moreno PT,
Cruz-Hipolito HE, Alcntara-de la Cruz R, Snchez-Gonzlez E, et al.
First confirmation and characterization of target and non-target site
resistance to glyphosate in Palmer amaranth (\emph{Amaranthus}
\emph{Palmeri}) from Mexico. Plant Physiology and Biochemistry.
2017;115: 212--218.
doi:\href{https://doi.org/10.1016/j.plaphy.2017.03.022}{10.1016/j.plaphy.2017.03.022}

\leavevmode\hypertarget{ref-varanasi_confirmation_2018}{}%
29. Varanasi VK, Brabham C, Norsworthy JK. Confirmation and
Characterization of Non--target site Resistance to Fomesafen in Palmer
amaranth (Amaranthus palmeri). Weed Science. 2018;66: 702--709.
doi:\href{https://doi.org/10.1017/wsc.2018.60}{10.1017/wsc.2018.60}

\leavevmode\hypertarget{ref-doyle_rapid_1987}{}%
30. Doyle JJ, Doyle JL. A rapid DNA isolation procedure for small
quantities of fresh leaf tissue. Phytochemical Bull. 1987;19: 11--15.
Available: \url{https://worldveg.tind.io/record/33886}

\leavevmode\hypertarget{ref-wuerffel_distribution_2015}{}%
31. Wuerffel RJ, Young JM, Lee RM, Tranel PJ, Lightfoot DA, Young BG.
Distribution of the G210 protoporphyrinogen oxidase mutation in Illinois
waterhemp (\emph{Amaranthus} \emph{Tuberculatus}) and an improved
molecular method for detection. Weed Science. 2015;63: 839--845.
doi:\href{https://doi.org/10.1614/WS-D-15-00037.1}{10.1614/WS-D-15-00037.1}

\leavevmode\hypertarget{ref-varanasi_statewide_2018}{}%
32. Varanasi VK, Brabham C, Norsworthy JK, Nie H, Young BG, Houston M,
et al. A Statewide survey of PPO-inhibitor resistance and the prevalent
target-site mechanisms in Palmer amaranth (\emph{Amaranthus}
\emph{Palmeri}) accessions from Arkansas. Weed Science. 2018;66:
149--158.
doi:\href{https://doi.org/10.1017/wsc.2017.68}{10.1017/wsc.2017.68}

\leavevmode\hypertarget{ref-chatham_multistate_2015}{}%
33. Chatham LA, Bradley KW, Kruger GR, Martin JR, Owen MDK, Peterson DE,
et al. A Multistate study of the association between glyphosate
resistance and EPSPS gene amplification in waterhemp (\emph{Amaranthus}
\emph{Tuberculatus}). Weed Science. 2015;63: 569--577.
doi:\href{https://doi.org/10.1614/WS-D-14-00149.1}{10.1614/WS-D-14-00149.1}

\leavevmode\hypertarget{ref-r_core_team_r:_2019}{}%
34. R: A Language and Environment for Statistical Computing
{[}Internet{]}. Vienna, Austria: R Foundation for Statistical Computing;
2019. Available: \url{https://www.R-project.org/}

\leavevmode\hypertarget{ref-breiman_random_2001}{}%
35. Breiman L. Random Forests. Machine Learning. 2001;45: 5--32.
doi:\href{https://doi.org/10.1023/A:1010933404324}{10.1023/A:1010933404324}

\leavevmode\hypertarget{ref-biau_random_2016}{}%
36. Biau G, Scornet E. A random forest guided tour. TEST. 2016;25:
197--227.
doi:\href{https://doi.org/10.1007/s11749-016-0481-7}{10.1007/s11749-016-0481-7}

\leavevmode\hypertarget{ref-langemeier_factors_2016}{}%
37. Langemeier CB, Robertson AE, Wang D, Jackson-Ziems TA, Kruger GR.
Factors affecting the development and severity of goss's bacterial wilt
and leaf blight of corn, caused by \emph{Clavibacter}
\emph{Michiganensis} subsp. \emph{Nebraskensis}. Plant Disease.
2016;101: 54--61.
doi:\href{https://doi.org/10.1094/PDIS-01-15-0038-RE}{10.1094/PDIS-01-15-0038-RE}

\leavevmode\hypertarget{ref-vieira_distribution_2018}{}%
38. Vieira BC, Samuelson SL, Alves GS, Gaines TA, Werle R, Kruger GR.
Distribution of glyphosate-resistant \emph{Amaranthus} spp. In Nebraska.
Pest Management Science. 2018;74: 2316--2324.
doi:\href{https://doi.org/10.1002/ps.4781}{10.1002/ps.4781}

\leavevmode\hypertarget{ref-gaines_gene_2010}{}%
39. Gaines TA, Zhang W, Wang D, Bukun B, Chisholm ST, Shaner DL, et al.
Gene amplification confers glyphosate resistance in \emph{Amaranthus}
\emph{Palmeri}. Proceedings of the National Academy of Sciences.
2010;107: 1029--1034.
doi:\href{https://doi.org/10.1073/pnas.0906649107}{10.1073/pnas.0906649107}

\leavevmode\hypertarget{ref-sammons_glyphosate_2014}{}%
40. Sammons RD, Gaines TA. Glyphosate resistance: State of knowledge.
Pest Management Science. 2014;70: 1367--1377.
doi:\href{https://doi.org/10.1002/ps.3743}{10.1002/ps.3743}

\leavevmode\hypertarget{ref-patterson_glyphosate_2018}{}%
41. Patterson EL, Pettinga DJ, Ravet K, Neve P, Gaines TA. Glyphosate
resistance and EPSPS gene duplication: Convergent evolution in multiple
plant species. J Hered. 2018;109: 117--125.
doi:\href{https://doi.org/10.1093/jhered/esx087}{10.1093/jhered/esx087}

\leavevmode\hypertarget{ref-remnant_gene_2013}{}%
42. Remnant EJ, Good RT, Schmidt JM, Lumb C, Robin C, Daborn PJ, et al.
Gene duplication in the major insecticide target site, Rdl, in
\emph{Drosophila} \emph{Melanogaster}. Proc Natl Acad Sci USA. 2013;110:
14705--14710.
doi:\href{https://doi.org/10.1073/pnas.1311341110}{10.1073/pnas.1311341110}

\leavevmode\hypertarget{ref-bass_gene_2011}{}%
43. Bass C, Field LM. Gene amplification and insecticide resistance.
Pest Management Science. 2011;67: 886--890.
doi:\href{https://doi.org/10.1002/ps.2189}{10.1002/ps.2189}

\leavevmode\hypertarget{ref-jugulam_tandem_2014}{}%
44. Jugulam M, Niehues K, Godar AS, Koo D-H, Danilova T, Friebe B, et
al. Tandem amplification of a chromosomal segment harboring
5-enolpyruvylshikimate-3-phosphate synthase locus confers glyphosate
resistance in \emph{Kochia} \emph{Scoparia}. Plant Physiology. 2014;166:
1200--1207.
doi:\href{https://doi.org/10.1104/pp.114.242826}{10.1104/pp.114.242826}

\leavevmode\hypertarget{ref-giacomini_variable_2019}{}%
45. Giacomini DA, Westra P, Ward SM. Variable Inheritance of Amplified
EPSPS Gene Copies in Glyphosate-Resistant Palmer Amaranth
(\emph{Amaranthus} \emph{Palmeri}). Weed Science. 2019;67: 176--182.
doi:\href{https://doi.org/10.1017/wsc.2018.65}{10.1017/wsc.2018.65}

\leavevmode\hypertarget{ref-malone_epsps_2016}{}%
46. Malone JM, Morran S, Shirley N, Boutsalis P, Preston C. EPSPS gene
amplification in glyphosate-resistant \emph{Bromus} \emph{Diandrus}.
Pest Management Science. 2016;72: 81--88.
doi:\href{https://doi.org/10.1002/ps.4019}{10.1002/ps.4019}

\leavevmode\hypertarget{ref-wiersma_gene_2015}{}%
47. Wiersma AT, Gaines TA, Preston C, Hamilton JP, Giacomini D, Robin
Buell C, et al. Gene amplification of
5-enol-pyruvylshikimate-3-phosphate synthase in glyphosate-resistant
\emph{Kochia} \emph{Scoparia}. Planta. 2015;241: 463--474.
doi:\href{https://doi.org/10.1007/s00425-014-2197-9}{10.1007/s00425-014-2197-9}

\leavevmode\hypertarget{ref-kumar_molecular_2015}{}%
48. Kumar V, Jha P, Giacomini D, Westra EP, Westra P. Molecular basis of
evolved resistance to glyphosate and acetolactate synthase-inhibitor
herbicides in Kochia (\emph{Kochia} \emph{Scoparia}) accessions from
Montana. Weed sci. 2015;63: 758--769.
doi:\href{https://doi.org/10.1614/WS-D-15-00021.1}{10.1614/WS-D-15-00021.1}

\leavevmode\hypertarget{ref-koo_extrachromosomal_2018}{}%
49. Koo D-H, Molin WT, Saski CA, Jiang J, Putta K, Jugulam M, et al.
Extrachromosomal circular DNA-based amplification and transmission of
herbicide resistance in crop weed \emph{Amaranthus} \emph{Palmeri}. Proc
Natl Acad Sci USA. 2018;115: 3332--3337.
doi:\href{https://doi.org/10.1073/pnas.1719354115}{10.1073/pnas.1719354115}

\leavevmode\hypertarget{ref-schafer_rhizosphere_2014}{}%
50. Schafer JR, Hallett SG, Johnson WG. Rhizosphere Microbial Community
Dynamics in Glyphosate-Treated Susceptible and Resistant Biotypes of
Giant Ragweed (Ambrosia trifida). Weed Science. 2014;62: 370--381.
doi:\href{https://doi.org/10.1614/WS-D-13-00164.1}{10.1614/WS-D-13-00164.1}

\leavevmode\hypertarget{ref-coburn_influence_2017}{}%
51. Coburn C. Influence of Experimental Methods on Herbicide Resistance
Confirmation. Ph.D dissertation, University of Wyoming. 2017.

\leavevmode\hypertarget{ref-jhala_confirmation_2014}{}%
52. Jhala AJ, Sandell LD, Rana N, Kruger GR, Knezevic SZ. Confirmation
and control of triazine and 4-hydroxyphenylpyruvate
dioxygenase-inhibiting herbicide-resistant Palmer amaranth
(\emph{Amaranthus} \emph{Palmeri}) in Nebraska. Weed Technology.
2014;28: 28--38.
doi:\href{https://doi.org/10.1614/WT-D-13-00090.1}{10.1614/WT-D-13-00090.1}

\leavevmode\hypertarget{ref-chahal_glyphosate-resistant_2017}{}%
53. Chahal PS, Varanasi VK, Jugulam M, Jhala AJ. Glyphosate-resistant
Palmer amaranth (\emph{Amaranthus} \emph{Palmeri}) in Nebraska:
Confirmation, EPSPS gene amplification, and response to POST corn and
soybean herbicides. Weed Technology. 2017;31: 80--93.
doi:\href{https://doi.org/10.1614/WT-D-16-00109.1}{10.1614/WT-D-16-00109.1}

\leavevmode\hypertarget{ref-andrade_weed_2017}{}%
54. Andrade JF, Satorre EH, Ermcora CM, Poggio SL. Weed communities
respond to changes in the diversity of crop sequence composition and
double cropping. Weed Research. 2017;57: 148--158.
doi:\href{https://doi.org/10.1111/wre.12251}{10.1111/wre.12251}

\leavevmode\hypertarget{ref-evans_managing_2016}{}%
55. Evans JA, Tranel PJ, Hager AG, Schutte B, Wu C, Chatham LA, et al.
Managing the evolution of herbicide resistance: Managing the evolution
of herbicide resistance. Pest Manag Sci. 2016;72: 74--80.
doi:\href{https://doi.org/10.1002/ps.4009}{10.1002/ps.4009}

\leavevmode\hypertarget{ref-beckie_selecting_2009}{}%
56. Beckie HJ, Reboud X. Selecting for weed resistance: Herbicide
rotation and mixture. Weed Technology. 2009;23: 363--370.
doi:\href{https://doi.org/10.1614/WT-09-008.1}{10.1614/WT-09-008.1}

\leavevmode\hypertarget{ref-strom_characterization_2019}{}%
57. Strom SA, Gonzini LC, Mitsdarfer C, Davis AS, Riechers DE, Hager AG.
Characterization of multiple herbicide--resistant waterhemp
(\emph{Amaranthus} \emph{Tuberculatus}) populations from Illinois to
VLCFA-inhibiting herbicides. Weed Science. 2019;67: 369--379.
doi:\href{https://doi.org/10.1017/wsc.2019.13}{10.1017/wsc.2019.13}

\leavevmode\hypertarget{ref-shergill_investigations_2018}{}%
58. Shergill LS, Barlow BR, Bish MD, Bradley KW. Investigations of 2,4-D
and multiple herbicide resistance in a Missouri waterhemp
(\emph{Amaranthus} \emph{Tuberculatus}) population. Weed Science.
2018;66: 386--394.
doi:\href{https://doi.org/10.1017/wsc.2017.82}{10.1017/wsc.2017.82}

\leavevmode\hypertarget{ref-tranel_target-site_2017}{}%
59. Tranel PJ, Wu C, Sadeque A. Target-site resistances to ALS and PPO
inhibitors are linked in waterhemp (\emph{Amaranthus}
\emph{Tuberculatus}). Weed Science. 2017;65: 4--8.
doi:\href{https://doi.org/10.1614/WS-D-16-00090.1}{10.1614/WS-D-16-00090.1}

\nolinenumbers


\end{document}

